.env.example

```diff
@@ -32,10 +32,10 @@ APP_URL=http://host.docker.internal:8080
 SAML_IDP_METADATA_PATH=config/idp-metadata-dev.xml
 
 # Background jobs
-# 24 * 60 * 60 * 1000 = 604800000 (24 hours)
+# 24 * 60 * 60 * 1000 = 604800000 (24 hours in milliseconds)
 EXPIRATION_REPEAT_FREQUENCY_MS=86400000
-# 7 * 24 * 60 * 60 * 1000 = 604800000 (7 days)
-JOB_REMOVAL_FREQUENCY_MS=604800000
+# 7 * 24 * 60 * 60 * 1000 = 604800 (7 days in seconds)
+JOB_REMOVAL_FREQUENCY_S=604800
```

Comment:
Above you use `_MS`, let's not mix the unit in this file. Change this to `_MS` too.

Reply:
Also, please update `DEPLOY.md` with whatever you add to the app's environment variables